### ISSUE TITLE
Documentation: TFH Dashboard Issue Template

### ISSUE THREAD

**Issue by leahcimto** [theforumhelpers/theforumhelpers.github.io#351]
**What does this affect?**
Issue templates

**Changes/Additions**
There should be an issue template for the dashboard

**Comment by BarelySmooth** [comment]
The dashboard isn't part of this repository. Feedback for the dashboard should have its own repo (when it eventually becomes public in the future)

**Comment by leahcimto** [comment]
True... so what you are saying is create a blank repo for the dashboard and when it becomes open source move it's code there? @Accio1

**Comment by BarelySmooth** [comment]
it would be a better idea to wait until the code is open-sourced (moving the code to a new repository wont preserve things like commit history)

**Comment by penguinmoose** [comment]
@BarelySmooth Right. We can put that repo in the same organization. But it needs to get open sourced first, of course

**Comment by leahcimto** [comment]
@Accio1 Is the code for the dashboard in a GH private repo somewhere or are you working on it somewhere else than github?

**Comment by penguinmoose** [comment]
@Accio1 If it's a private repo, it shouldn't be hard to make it public and move it here.

**Comment by Accio1** [comment]
> @Accio1 Is the code for the dashboard in a GH private repo somewhere or are you working on it somewhere else than github?

I'm working on it off GitHub using Git so it does have a Git history. However, I'm still not quite ready to open source it. In the meantime, I agree that there should be an issue template for dashboard issues. It should include the policy that any security issues with the dashboard are not shared publicly and rather posted in the comments of one of my older Scratch projects.

@leahcimto or @penguinmoose would either of you be interested in drafting the issue template?

**Comment by leahcimto** [comment]
@Accio1 Would it be okay if I did? @penguinmoose if you want to do it that's fine too!

**Comment by Accio1** [comment]
@leahcimto, yes you can go ahead and work on it. @penguinmoose left a comment in TFH studio saying they would be inactive for a few days starting today so they won't be able to work on it yet anyway.

**Comment by penguinmoose** [comment]
@Accio1 Actually, I’d love to work on it. I’m more active now.

**Comment by Accio1** [comment]
@penguinmoose you'll have to ask leahcimto, he was already assigned and may have started working on it already.

**Comment by penguinmoose** [comment]
@leahcimto Is it okay if I work on it? If you are already working on it, no problem.

**Comment by leahcimto** [comment]
@Accio1 should there be separate issue templates for dashboard bugs and dashboard feature requests, or should they be the same?

**Comment by penguinmoose** [comment]
@leahcimto Thanks!

Wait, so the dashboard code and issues relating to that would be put in this repo? If so, here’s a draft:

 _If this is about a security vulnerability, please do not post it publicly here. Instead, comment about it on one of -Accio-‘s old projects._

### Is this issue related to a dashboard problem?
_If so, please describe the problem here._

### What feature, solution, or change would you like to be implemented?
_Please describe the change thoroughly._

### Alternatives I’ve considered
_List some alternatives to the proposed solution here._

### Additional context
_If you have additional context (such as screenshots), put them here._

**Comment by leahcimto** [comment]
That looks great! One thing you might want to do is clarify "Scratch project" in the disclaimer at the top.

**Comment by penguinmoose** [comment]
@leahcimto Ok, fixed.
